### DISCUSSION TITLE
اپلیکیشن یا وبسایت اعتراضات/اجتماعات یاب روی نقشه براساس بلوتوث و جی پی اس

### DISCUSSION THREAD

**Discussion by hajiparvaneh** [WeAreMahsaAmini/Core#451] [Ideas]
اپلیکیشن یا وبسایت اعتراضات/اجتماعات یاب روی نقشه براساس بلوتوث و جی پی اس

**Comment by arashi-dev** [comment]
این اتفاقا تو یکی از دیسکاشن ها براش بحث شد. و یکی از دوستان گفت که همچین چیزی بیشتر بدرد سرکوبگرا میخوره تا مردم. اونا اینجوری راحت میدونن چند نفر کجا رفتن و باید چقدر نیرو بفرستن

**Comment by tcf-admin** [comment]
درود، 
به این پاسخ ما در موردی مشابه توجه بفرمایید: 
https://github.com/WeAreMahsaAmini/Core/discussions/466#discussioncomment-4380891
